multihost_test/bz_automation/test_automation.py

```diff
@@ -310,3 +310,29 @@ def test_pam_access_account(self, multihost, create_localusers, bkp_pam_config):
         client.run_command("sh /tmp/authentication.sh")
         log_str = multihost.client[0].get_file_contents("/var/log/secure").decode('utf-8')
         assert "LOCAL" not in log_str
+
+    def test_cve_pam_access(self, multihost, bkp_pam_config, create_localusers):
```

Comment:
Provide a proper name to what we are testing here. This isn't the first CVE for pam_access, and it won't be the last one